setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name="tinman",
-      version          = 0.1,
+      version          = __import__('tinman').__version__,
```

Comment:
Based on our experience with Steem Python, do you think this will create an issue?

Reply:
I think the steem-python thing was a circle-ci issue.